cronner/kronjob_util.py

```diff
@@ -6,12 +6,11 @@ class KronjobValidationException(Exception):
 
 
 def format_kronjob_entry(entry):
-    return {
+    return dict(**{
```

Comment:
I think you'll need to remove the `**` here in order to support older python versions

Reply:
thanks yep just did